modules/javafx.controls/src/test/java/test/javafx/scene/control/TableViewTest.java

```diff
@@ -3871,22 +3929,22 @@ private void test_rt_37538(boolean callCNextOnce, boolean callCNextInLoop) {
         sl.dispose();
     }
 
-    @Ignore("Fix not yet developed for TableView")
+    @Disabled("Fix not yet developed for TableView")
```

Comment:
`@Test` was placed on its own line elsewhere but not in this file?

Reply:
Hmm, I suggested @EstelonAgarwaen to use the IntelliJ Junit 5 Migration functionality. So looks like it is not always putting `@Test` in a new line. Only when nothing is above it seems?